backend/btrixcloud/models.py

```diff
@@ -1180,7 +1180,6 @@ class S3Storage(BaseModel):
     secret_key: str
     access_endpoint_url: str
     region: str = ""
-    use_access_for_presign: bool = True
```

Comment:
We should make sure to mention this deprecation in the release changelog or documentation